task3/script.js

```diff
@@ -11,5 +11,5 @@ for (let i =1; i <=2; i++) {
     ul.appendChild(li).appendChild(a).insertAdjacentElement('afterbegin', img);
 }
 
-const firstChild = ul.firstChild;
-
+ul.firstChild.querySelector('img').src = "/img/bookmark.png";
+ul.lastChild.querySelector('img').src = "/img/message.png";
```

Comment:
[IMO]
I like the way you used firstChild and lastChild to get the first and last elements, and changed the src attribute!
Another way is to use the src attribute of img as array data and output it using forEach or other methods. You may have adopted the method from assignment 4, so please ask others for their opinions.

Reply:
Arrays make the code simpler when used in these situations. I understand.

Thank you for the review.
I would like to change the code to be easy to read and simple.